README.md

```diff
@@ -29,9 +29,15 @@ Version 2 of the Envoy API will evolve existing APIs and introduce new APIs to:
 
 ## Status
 
-Draft work-in-progress. Input is welcome via issue filing. Small, localized PRs
-are also welcome, but any major changes or suggestions should be coordinated in
-a tracking issue with the authors.
+The LDS/CDS/EDS/RDS APIs are now frozen and will maintain backwards
```

Comment:
@PiotrSikora you still have time if you want to make some changes that wouldn't be just straight up field additions. Can you do it soon?